api/server.go

```diff
@@ -98,14 +99,25 @@ func (s *Server) walletAddressHandler(w http.ResponseWriter, req *http.Request,
 
 func (s *Server) walletAddressesHandler(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
 	addresses := s.w.Addresses()
-	offset, limit := offsetLimit(req.URL.Query(), len(addresses))
-	writeJSON(w, WalletAddresses(addresses[offset:limit]))
+	begin, end := beginEnd(req.URL.Query(), len(addresses))
+	writeJSON(w, WalletAddresses(addresses[begin:end]))
 }
 
 func (s *Server) walletTransactionsHandler(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
-	transactions := s.w.Transactions()
-	offset, limit := offsetLimit(req.URL.Query(), len(transactions))
-	writeJSON(w, transactions[offset:limit])
+	q := req.URL.Query()
+	since := q.Get("since")
+	sinceParsed, err := strconv.ParseInt(since, 10, 64)
```

Comment:
RFC3339 is probably a better choice than Unix timestamps:
```go
var since time.Time
if v := req.FormValue("since"); v != "" {
    t, err := time.Parse(time.RFC3339, req.FormValue("since")
    if err != nil {
        http.Error(w, "invalid since value: " + err.Error(), http.StatusBadRequest)
        return
    }
}